### PR TITLE
Add in progress alive evil reporting

### PR DESCRIPTION
**Pull request by mbj**

### CODE REVIEW

File: lib/mutant/reporter/cli.rb [hunk 1]
```diff
@@ -20,6 +20,14 @@ def self.build(output)
         )
       end
 
+      # Report alive evil mutation
+      #
+      # @return [self]
+      def alive_evil(mutation_result)
+        write(format.alive_evil(mutation_resutl))
```
Comment (snusnu) [inline lib/mutant/reporter/cli.rb]: @mbj `s/mutation_resutl/mutation_result/`

File: spec/unit/mutant/runner/sink_spec.rb [hunk 2]
```diff
@@ -133,40 +165,52 @@
...
+          expect(apply).to eql(false)
+        end
       end
 
       context 'one result' do
         include_context 'one result'
 
         context 'when result is successful' do
-          it { should be(false) }
+          it 'returns expected value' do
```
Comment (chaubunhtha) [inline spec/unit/mutant/runner/sink_spec.rb:177]: p

File: spec/unit/mutant/runner/sink_spec.rb [hunk 3]
```diff
@@ -133,40 +165,52 @@
...
+            expect(apply).to eql(false)
```
Comment (chaubunhtha) [inline spec/unit/mutant/runner/sink_spec.rb:178]: n

File: spec/unit/mutant/runner/sink_spec.rb [hunk 4]
```diff
@@ -133,40 +165,52 @@
...
+          end
         end
```
Comment (chaubunhtha) [inline spec/unit/mutant/runner/sink_spec.rb:180]: p

File: spec/unit/mutant/runner/sink_spec.rb [hunk 5]
```diff
@@ -133,40 +165,52 @@
...
 
         context 'when result failed' do
           with(:mutation_a_test_result) { { passed: true } }
 
-          it { should be(true) }
+          it 'returns expected value' do
```
Comment (chaubunhtha) [inline spec/unit/mutant/runner/sink_spec.rb:185]: n

File: spec/unit/mutant/runner/sink_spec.rb [hunk 6]
```diff
@@ -133,40 +165,52 @@
...
+            expect(apply).to eql(true)
```
Comment (chaubunhtha) [inline spec/unit/mutant/runner/sink_spec.rb:186]: p

File: spec/unit/mutant/runner/sink_spec.rb [hunk 7]
```diff
@@ -133,40 +165,52 @@
...
+          end
```
Comment (chaubunhtha) [inline spec/unit/mutant/runner/sink_spec.rb:187]: n

File: spec/unit/mutant/runner/sink_spec.rb [hunk 8]
```diff
@@ -133,40 +165,52 @@
...
         end
       end
 
       context 'two results' do
         include_context 'two results'
 
         context 'when results are successful' do
-          it { should be(false) }
+          it 'returns expected value' do
```
Comment (chaubunhtha) [inline spec/unit/mutant/runner/sink_spec.rb:195]: p

File: spec/unit/mutant/runner/sink_spec.rb [hunk 9]
```diff
@@ -133,40 +165,52 @@
...
+            expect(apply).to eql(false)
```
Comment (chaubunhtha) [inline spec/unit/mutant/runner/sink_spec.rb:196]: n

File: spec/unit/mutant/runner/sink_spec.rb [hunk 10]
```diff
@@ -133,40 +165,52 @@
...
+          end
```
Comment (chaubunhtha) [inline spec/unit/mutant/runner/sink_spec.rb:197]: p

File: spec/unit/mutant/runner/sink_spec.rb [hunk 11]
```diff
@@ -133,40 +165,52 @@
...
         end
 
         context 'when first result is unsuccessful' do
           with(:mutation_a_test_result) { { passed: true } }
 
-          it { should be(true) }
+          it 'returns expected value' do
```
Comment (chaubunhtha) [inline spec/unit/mutant/runner/sink_spec.rb:203]: n

File: spec/unit/mutant/runner/sink_spec.rb [hunk 12]
```diff
@@ -133,40 +165,52 @@
...
+            expect(apply).to eql(true)
```
Comment (chaubunhtha) [inline spec/unit/mutant/runner/sink_spec.rb:204]: p

File: spec/unit/mutant/runner/sink_spec.rb [hunk 13]
```diff
@@ -133,40 +165,52 @@
...
+          end
```
Comment (chaubunhtha) [inline spec/unit/mutant/runner/sink_spec.rb:205]: n

File: spec/unit/mutant/runner/sink_spec.rb [hunk 14]
```diff
@@ -133,40 +165,52 @@
...
         end
 
         context 'when second result is unsuccessful' do
           with(:mutation_b_test_result) { { passed: true } }
 
-          it { should be(true) }
+          it 'returns expected value' do
```
Comment (chaubunhtha) [inline spec/unit/mutant/runner/sink_spec.rb:211]: p

File: spec/unit/mutant/runner/sink_spec.rb [hunk 15]
```diff
@@ -133,40 +165,52 @@
...
+            expect(apply).to eql(true)
```
Comment (chaubunhtha) [inline spec/unit/mutant/runner/sink_spec.rb:212]: n

File: spec/unit/mutant/runner/sink_spec.rb [hunk 16]
```diff
@@ -133,40 +165,52 @@
...
+          end
```
Comment (chaubunhtha) [inline spec/unit/mutant/runner/sink_spec.rb:213]: p

File: spec/unit/mutant/runner_spec.rb [hunk 17]
```diff
@@ -60,7 +60,7 @@
         jobs:               1,
         mutex:              mutex,
         processor:          processor,
-        sink:               Mutant::Runner::Sink.new(env),
+        sink:               Mutant::Runner::Sink.new(env, reporter),
```
Comment (chaubunhtha) [inline spec/unit/mutant/runner_spec.rb:63]: n

**Review by chaubunhtha — COMMENTED** [review]
.md

**Comment by mbj** [comment]
@snusnu I'm closing this as too old. I wish to implement this change, but on a newer base. Possibly in the new main process.

**Pull request closed without merging**